.travis.yml

```diff
@@ -2,37 +2,26 @@ language: php
 
 sudo: false
 
-php:
-  - 7.0
-  - 7.1
-
-env:
-  - SYMFONY_VERSION=~2.8.0
-  - SYMFONY_VERSION=~3.0
-  - SYMFONY_VERSION=~4.0
-  - DEPENDENCIES=beta
-
 matrix:
   fast_finish: true
-  exclude:
-    - php: 5.6
-      env: DEPENDENCIES=beta
-    - php: 7.0
-      env: SYMFONY_VERSION=~4.0
-    - php: 7.0
-      env: DEPENDENCIES=beta
+  include:
+    - php: 7.2
+      env: SYMFONY_VERSION=~2.8.0
+    - php: 7.3
+      env: SYMFONY_VERSION=^3.0
+    - php: 7.3
+      env: SYMFONY_VERSION=^4.0
 
 cache:
   directories:
     - $HOME/.composer/cache
 
 before_install:
-  - if [ "$DEPENDENCIES" = "beta" ]; then composer config minimum-stability beta; fi;
+  - composer self-update --preview
 
-before_script:
-  - composer selfupdate
+install:
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
-  - composer update --prefer-dist $COMPOSER_FLAGS
+  - composer update --prefer-source --no-interaction $COMPOSER_FLAGS
```

Comment:
Changing to `prefer-source` as tests are relying on `Symfony\Bridge\Twig\Tests\Extension\Fixtures\StubTranslator` which are not anymore available when using dist versions, as `.gitattributes` was added and removed the Tests folder (`/Tests export-ignore` rule) from package.